clients/src/main/java/org/apache/kafka/clients/consumer/internals/FetchMetricsManager.java

```diff
@@ -181,6 +195,67 @@ void maybeUpdateAssignment(SubscriptionState subscription) {
         }
     }
 
+    @Deprecated // To be removed in Kafka 5.0 release.
+    private void maybeRecordDeprecatedBytesFetched(String name, String topic, int bytes) {
```

Comment:
It would be useful to follow the approach used in https://github.com/apache/kafka/pull/11302/files to add "deprecated" in the description of the metric name and update the ops doc about the deprecation.

Reply:
As the metrics with a specfic tag has been deprectaed hence it's hard to specify same metric as deprecated. hence I have added `Note` to consumer topic metrics. For consumer metrics ops need not to ne updated as ops.html uses metrics description to generate details on kafka-site, I have verified that the added `Note` can be viewed on kafka-site locally.